cmd/syncer/main.go

```diff
@@ -65,6 +68,12 @@ func main() {
 		log.Errorf("failed to initialize the configInfo. Err: %+v", err)
 		os.Exit(1)
 	}
+	// Initialize Kubernetes feature state config instance
+	k8sFeatureStateConfig, err := commonco.GetContainerOrchestratorInterface(ctx, common.Kubernetes, configInfo.Cfg.FeatureStatesConfig)
```

Comment:
shouldn't this be within the `run` function? We only want one instance of the admission webhook to be the leader right?

Reply:
No, we want all webhook instances to be up and running regardless of leader election. 
If we create multiple replicas of our controller pod, the service bind to all those pods and if a validation request comes to the pod where the webhook is not running, the operation will be disallowed because the webhook is not accessible.

Reply:
>If we create multiple replicas of our controller pod
But this is not a supported model right for our CSI controller?

>  if a validation request comes to the pod where the webhook is not running, the operation will be disallowed because the webhook is not accessible.
If guarded by leader election, the validation request will only go to the webhook where the leader is running. isn't that true?

Reply:
> If guarded by leader election, the validation request will only go to the webhook where the leader is running. isn't that true?

No this is not true. The validation request does not honor or look who is a leader. 
A review request can be sent to any endpoint address bound to the service.